### PR TITLE
[circleci] migrating to v2

### PR DESCRIPTION
**Pull request by truthbk**

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -1,26 +0,0 @@
...
-    - "~/.chefdk"
-
-test:
-  pre:
-    - eval "$(/opt/chefdk/bin/chef shell-init bash)"
-  override:
-    - chef --version
-    - chef gem query -d chefspec rubocop foodcritic rake kitchen-docker
-    - chef exec rake circle:
-        parallel: true
```
Comment (hush-hush) [inline circle.yml]: Do we need to add `parallel` and `timeout` to the new circleci file ?

Comment (truthbk) [reply circle.yml]: I couldn't find the way to add the directives to the job in `v2`. I'll take another look.

Comment (hush-hush) [reply circle.yml]: OK, we should just double check what they did in V1 and that we no longer need them with V2.

Comment (truthbk) [reply circle.yml]: setting parallelism would not help in circleci v2. It didn't really accomplish much in v1 anyways.

**Pull request merged**